src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -16,11 +16,14 @@ import MegaMenu from '@department-of-veterans-affairs/formation/MegaMenu';
 
 export function flagCurrentPageInTopLevelLinks(
   links = [],
+  href = window.location.href,
   pathName = window.location.pathname,
 ) {
   return links.map(
     link =>
-      pathName.endsWith(link.href) ? { ...link, currentPage: true } : link,
+      pathName.endsWith(link.href) || href === link.href
```

Comment:
this determines which link is the "currentPage" which triggers an additional style to be added to the list item in the top nav of the mega menu.

since we moved to using fully qualified urls for staging / production environments, checking just the pathName is not sufficient. Added a check against the full href as well. 

- on dev, the `link.href` will be something like `/facilities` which will pass the `pathName.endsWith` check
- on staging / prod, the `link.href` will something like 'https://preview.va.gov/facilities which will pass the `href == = link.href` check